notes/CBA_Settings.txt
// 社群基础模组
force force cba_disposable_dropUsedLauncher = 2;
force force cba_disposable_replaceDisposableLauncher = true;
force force cba_network_loadoutValidation = 0;

// ACE夜视镜
force force ace_nightvision_aimDownSightsBlur = 0;
force force ace_nightvision_disableNVGsWithSights = false;
force force ace_nightvision_effectScaling = 0.1;
force force ace_nightvision_fogScaling = 0;
force force ace_nightvision_noiseScaling = 0.1;
force force ace_nightvision_shutterEffects = true;

// ACE虚拟军火库
force force ace_arsenal_allowDefaultLoadouts = true;
force force ace_arsenal_allowSharedLoadouts = true;
force force ace_arsenal_camInverted = false;
force force ace_arsenal_enableIdentityTabs = true;
force force ace_arsenal_enableModIcons = true;
force force ace_arsenal_EnableRPTLog = false;
force force ace_arsenal_fontHeight = 4.5;

// ACE 互动选单
force force ace_gestures_showOnInteractionMenu = 2;
force force ace_interact_menu_actionOnKeyRelease = true;
force force ace_interact_menu_addBuildingActions = false;
force force ace_interact_menu_alwaysUseCursorInteraction = false;
force force ace_interact_menu_alwaysUseCursorSelfInteraction = true;
force force ace_interact_menu_consolidateSingleChild = false;
force force ace_interact_menu_cursorKeepCentered = false;
force force ace_interact_menu_cursorKeepCenteredSelfInteraction = true;
force force ace_interact_menu_menuAnimationSpeed = 0;
force force ace_interact_menu_menuBackground = 0;
force force ace_interact_menu_menuBackgroundSelf = 0;
force force ace_interact_menu_shadowSetting = 2;
force force ace_interact_menu_textSize = 2;
force force ace_interact_menu_useListMenu = false;
force force ace_interact_menu_useListMenuSelf = false;

// ACE 互动
force force ace_interaction_disableNegativeRating = true;
force force ace_interaction_enableGroupRenaming = true;
force force ace_interaction_enableMagazinePassing = true;
force force ace_interaction_enableTeamManagement = true;
force force ace_interaction_enableWeaponAttachments = true;
force force ace_interaction_interactWithTerrainObjects = true;

// ACE 使用者介面
force force ace_ui_allowSelectiveUI = true;
force force ace_ui_ammoCount = false;
force force ace_ui_ammoType = true;
force force ace_ui_commandMenu = true;
force force ace_ui_enableSpeedIndicator = true;
force force ace_ui_firingMode = true;
force force ace_ui_groupBar = false;
force force ace_ui_gunnerAmmoCount = true;
force force ace_ui_gunnerAmmoType = true;
force force ace_ui_gunnerFiringMode = true;
force force ace_ui_gunnerLaunchableCount = true;
force force ace_ui_gunnerLaunchableName = true;
force force ace_ui_gunnerMagCount = true;
force force ace_ui_gunnerWeaponLowerInfoBackground = true;
force force ace_ui_gunnerWeaponName = true;
force force ace_ui_gunnerWeaponNameBackground = true;
force force ace_ui_gunnerZeroing = true;
force force ace_ui_magCount = true;
force force ace_ui_soldierVehicleWeaponInfo = true;
force force ace_ui_staminaBar = true;
force force ace_ui_stance = true;
force force ace_ui_throwableCount = true;
force force ace_ui_throwableName = true;
force force ace_ui_vehicleAltitude = true;
force force ace_ui_vehicleCompass = true;
force force ace_ui_vehicleDamage = true;
force force ace_ui_vehicleFuelBar = true;
force force ace_ui_vehicleInfoBackground = true;
force force ace_ui_vehicleName = true;
force force ace_ui_vehicleNameBackground = true;
force force ace_ui_vehicleRadar = true;
force force ace_ui_vehicleSpeed = true;
force force ace_ui_weaponLowerInfoBackground = true;
force force ace_ui_weaponName = true;
force force ace_ui_weaponNameBackground = true;
force force ace_ui_zeroing = true;

// ACE 俘虏
force force ace_captives_allowHandcuffOwnSide = false;
force force ace_captives_allowSurrender = true;
force force ace_captives_requireSurrender = 2;
force force ace_captives_requireSurrenderAi = false;

// ACE 先进弹道系统
force force ace_advanced_ballistics_ammoTemperatureEnabled = true;
force force ace_advanced_ballistics_barrelLengthInfluenceEnabled = true;
force force ace_advanced_ballistics_bulletTraceEnabled = true;
force force ace_advanced_ballistics_enabled = true;
force force ace_advanced_ballistics_muzzleVelocityVariationEnabled = true;
force force ace_advanced_ballistics_simulationInterval = 0.05;

// ACE 切换单位
force force ace_switchunits_enableSafeZone = true;
force force ace_switchunits_enableSwitchUnits = false;
force force ace_switchunits_safeZoneRadius = 100;
force force ace_switchunits_switchToCivilian = false;
force force ace_switchunits_switchToEast = false;
force force ace_switchunits_switchToIndependent = false;
force force ace_switchunits_switchToWest = false;

// ACE 医疗系统
force force ace_medical_ai_enabledFor = 2;
force force ace_medical_AIDamageThreshold = 0.01;
force force ace_medical_bleedingCoefficient = 0.5;
force force ace_medical_blood_bloodLifetime = 300;
force force ace_medical_blood_enabledFor = 2;
force force ace_medical_blood_maxBloodObjects = 500;
force force ace_medical_deathChance = 0.25;
force force ace_medical_enableVehicleCrashes = true;
force force ace_medical_fatalDamageSource = 2;
force force ace_medical_feedback_bloodVolumeEffectType = 2;
force force ace_medical_feedback_enableHUDIndicators = true;
force force ace_medical_feedback_painEffectType = 2;
force force ace_medical_fractureChance = 0.15;
force force ace_medical_fractures = 2;
force force ace_medical_gui_enableActions = 0;
force force ace_medical_gui_enableMedicalMenu = 1;
force force ace_medical_gui_enableSelfActions = true;
force force ace_medical_gui_interactionMenuShowTriage = 1;
force force ace_medical_gui_maxDistance = 5;
force force ace_medical_gui_openAfterTreatment = true;
force force ace_medical_ivFlowRate = 2.5;
force force ace_medical_limping = 1;
force force ace_medical_painCoefficient = 0.8;
force force ace_medical_painUnconsciousChance = 0.15;
force force ace_medical_playerDamageThreshold = 2.5;
force force ace_medical_spontaneousWakeUpChance = 1;
force force ace_medical_spontaneousWakeUpEpinephrineBoost = 3;
force force ace_medical_statemachine_AIUnconsciousness = false;
force force ace_medical_statemachine_cardiacArrestBleedoutEnabled = true;
force force ace_medical_statemachine_cardiacArrestTime = 300;
force force ace_medical_statemachine_fatalInjuriesAI = 0;
force force ace_medical_statemachine_fatalInjuriesPlayer = 1;
force force ace_medical_treatment_advancedBandages = 2;
force force ace_medical_treatment_advancedDiagnose = 1;
force force ace_medical_treatment_advancedMedication = true;
force force ace_medical_treatment_allowBodyBagUnconscious = false;
force force ace_medical_treatment_allowLitterCreation = true;
force force ace_medical_treatment_allowSelfIV = 1;
force force ace_medical_treatment_allowSelfPAK = 1;
force force ace_medical_treatment_allowSelfStitch = 1;
force force ace_medical_treatment_allowSharedEquipment = 0;
force force ace_medical_treatment_clearTrauma = 1;
force force ace_medical_treatment_consumePAK = 1;
force force ace_medical_treatment_consumeSurgicalKit = 0;
force force ace_medical_treatment_convertItems = 0;
force force ace_medical_treatment_cprSuccessChanceMax = 1;
force force ace_medical_treatment_cprSuccessChanceMin = 0.5;
force force ace_medical_treatment_holsterRequired = 0;
force force ace_medical_treatment_litterCleanupDelay = 180;
force force ace_medical_treatment_locationEpinephrine = 0;
force force ace_medical_treatment_locationIV = 0;
force force ace_medical_treatment_locationPAK = 3;
force force ace_medical_treatment_locationsBoostTraining = true;
force force ace_medical_treatment_locationSurgicalKit = 0;
force force ace_medical_treatment_maxLitterObjects = 100;
force force ace_medical_treatment_medicEpinephrine = 0;
force force ace_medical_treatment_medicIV = 0;
force force ace_medical_treatment_medicPAK = 2;
force force ace_medical_treatment_medicSurgicalKit = 1;
force force ace_medical_treatment_timeCoefficientPAK = 1;
force force ace_medical_treatment_treatmentTimeAutoinjector = 2.5;
force force ace_medical_treatment_treatmentTimeBodyBag = 5;
force force ace_medical_treatment_treatmentTimeCPR = 10;
force force ace_medical_treatment_treatmentTimeIV = 7.5;
force force ace_medical_treatment_treatmentTimeSplint = 5;
force force ace_medical_treatment_treatmentTimeTourniquet = 2.5;
force force ace_medical_treatment_woundReopenChance = 1;
force force ace_medical_treatment_woundStitchTime = 3;

// ACE 后勤设定
force force ace_cargo_enable = true;
force force ace_cargo_enableRename = true;
force force ace_cargo_loadTimeCoefficient = 5;
force force ace_cargo_openAfterUnload = 0;
force force ace_cargo_paradropTimeCoefficent = 2.5;
force force ace_rearm_distance = 25;
force force ace_rearm_level = 1;
force force ace_rearm_supply = 0;
force force ace_refuel_hoseLength = 25;
force force ace_refuel_rate = 1;
force force ace_repair_addSpareParts = false;
force force ace_repair_autoShutOffEngineWhenStartingRepair = true;
force force ace_repair_consumeItem_toolKit = 0;
force force ace_repair_displayTextOnRepair = true;
force force ace_repair_engineerSetting_fullRepair = 1;
force force ace_repair_engineerSetting_repair = 1;
force force ace_repair_engineerSetting_wheel = 0;
force force ace_repair_fullRepairLocation = 3;
force force ace_repair_fullRepairRequiredItems = ["ace_repair_anyToolKit"];
force force ace_repair_locationsBoostTraining = true;
force force ace_repair_miscRepairRequiredItems = ["ace_repair_anyToolKit"];
force force ace_repair_repairDamageThreshold = 0.75;
force force ace_repair_repairDamageThreshold_engineer = 0.8;
force force ace_repair_wheelRepairRequiredItems = [];

// ACE 听力设定
force force ace_hearing_autoAddEarplugsToUnits = false;
force force ace_hearing_disableEarRinging = false;
force force ace_hearing_earplugsVolume = 0.5;
force force ace_hearing_enableCombatDeafness = true;
force force ace_hearing_enabledForZeusUnits = false;
force force ace_hearing_unconsciousnessVolume = 0.3;

// ACE 地图标识器
force force ace_map_gestures_allowCurator = true;
force force ace_map_gestures_allowSpectator = true;
force force ace_map_gestures_briefingMode = 0;
force force ace_map_gestures_enabled = true;
force force ace_map_gestures_interval = 0.03;
force force ace_map_gestures_maxRange = 50;
force force ace_map_gestures_maxRangeCamera = 50;
force force ace_map_gestures_onlyShowFriendlys = true;

// ACE 地图
force force ace_map_BFT_Enabled = true;
force force ace_map_BFT_HideAiGroups = true;
force force ace_map_BFT_Interval = 3;
force force ace_map_BFT_ShowPlayerNames = true;
force force ace_map_DefaultChannel = 3;
force force ace_map_mapGlow = true;
force force ace_map_mapIllumination = true;
force force ace_map_mapLimitZoom = false;
force force ace_map_mapShake = true;
force force ace_map_mapShowCursorCoordinates = true;
force force ace_markers_moveRestriction = 0;
force force ace_markers_timestampEnabled = true;
force force ace_markers_timestampFormat = "HH:MM";
force force ace_markers_timestampHourFormat = 24;

// ACE 坐下功能
force force acex_sitting_enable = true;

// ACE 天气
force force ace_weather_enabled = true;
force force ace_weather_showCheckAirTemperature = true;
force force ace_weather_updateInterval = 90;
force force ace_weather_windSimulation = true;

// ACE 宙斯
force force ace_zeus_autoAddObjects = false;
force force ace_zeus_canCreateZeus = 0;
force force ace_zeus_radioOrdnance = false;
force force ace_zeus_remoteWind = false;
force force ace_zeus_revealMines = 0;
force force ace_zeus_zeusAscension = false;
force force ace_zeus_zeusBird = false;

// ACE 导弹挂架
force force ace_pylons_enabledForZeus = true;
force force ace_pylons_enabledFromAmmoTrucks = true;
force force ace_pylons_rearmNewPylons = false;
force force ace_pylons_requireEngineer = false;
force force ace_pylons_requireToolkit = true;
force force ace_pylons_searchDistance = 25;
force force ace_pylons_timePerPylon = 3;

// ACE 干粮
force force acex_field_rations_affectAdvancedFatigue = true;
force force acex_field_rations_enabled = false;
force force acex_field_rations_hudShowLevel = 0;
force force acex_field_rations_hudTransparency = -1;
force force acex_field_rations_hudType = 1;
force force acex_field_rations_hungerSatiated = 1;
force force acex_field_rations_terrainObjectActions = true;
force force acex_field_rations_thirstQuenched = 1;
force force acex_field_rations_timeWithoutFood = 2;
force force acex_field_rations_timeWithoutWater = 2;
force force acex_field_rations_waterSourceActions = 2;

// ACE 快速搭乘
force force ace_quickmount_distance = 5;
force force ace_quickmount_enabled = true;
force force ace_quickmount_speed = 25;

// ACE 手榴弹
force force ace_grenades_convertExplosives = true;

// ACE 护目镜
force force ace_goggles_effects = 2;
force force ace_goggles_showClearGlasses = true;
force force ace_goggles_showInThirdPerson = false;

// ACE 指向标记器
force force ace_finger_enabled = true;
force force ace_finger_indicatorForSelf = true;
force force ace_finger_maxRange = 50;

// ACE 旁观者
force force ace_spectator_enableAI = true;
force force ace_spectator_maxFollowDistance = 25;
force force ace_spectator_restrictModes = 0;
force force ace_spectator_restrictVisions = 0;

// ACE 无头客户端
force force acex_headless_delay = 15;
force force acex_headless_enabled = false;
force force acex_headless_endMission = 0;
force force acex_headless_log = false;
force force acex_headless_transferLoadout = 1;

// ACE 模拟碎片
force force ace_frag_enabled = true;
force force ace_frag_maxTrack = 10;
force force ace_frag_maxTrackPerFrame = 10;
force force ace_frag_reflectionsEnabled = true;
force force ace_frag_spallEnabled = true;

// ACE 武器按键
force force ace_common_persistentLaserEnabled = true;
force force ace_laserpointer_enabled = true;
force force ace_reload_displayText = true;
force force ace_reload_showCheckAmmoSelf = true;
force force ace_weaponselect_displayText = true;

// ACE 殉爆效果
force force ace_cookoff_ammoCookoffDuration = 0.5;
force force ace_cookoff_enable = 2;
force force ace_cookoff_enableAmmobox = true;
force force ace_cookoff_enableAmmoCookoff = true;
force force ace_cookoff_enableFire = true;
force force ace_cookoff_probabilityCoef = 0.5;

// ACE 火炮
force force ace_artillerytables_advancedCorrections = true;
force force ace_artillerytables_disableArtilleryComputer = false;
force force ace_mk6mortar_airResistanceEnabled = true;
force force ace_mk6mortar_allowCompass = true;
force force ace_mk6mortar_allowComputerRangefinder = true;
force force ace_mk6mortar_useAmmoHandling = false;

// ACE 炸药
force force ace_explosives_customTimerDefault = 30;
force force ace_explosives_customTimerMax = 900;
force force ace_explosives_customTimerMin = 5;
force force ace_explosives_explodeOnDefuse = true;
force force ace_explosives_punishNonSpecialists = true;
force force ace_explosives_requireSpecialist = false;

// ACE 玩家名字
force force ace_nametags_ambientBrightnessAffectViewDist = 1;
force force ace_nametags_playerNamesMaxAlpha = 1;
force force ace_nametags_playerNamesViewDistance = 50;
force force ace_nametags_showCursorTagForVehicles = true;
force force ace_nametags_showNamesForAI = false;
force force ace_nametags_showPlayerNames = 1;
force force ace_nametags_showPlayerRanks = false;
force force ace_nametags_showSoundWaves = 1;
force force ace_nametags_showVehicleCrewInfo = true;
force force ace_nametags_tagSize = 2;

// ACE 瞄准镜
force force ace_scopes_correctZeroing = true;
force force ace_scopes_deduceBarometricPressureFromTerrainAltitude = true;
force force ace_scopes_defaultZeroRange = 100;
force force ace_scopes_enabled = true;
force force ace_scopes_forceUseOfAdjustmentTurrets = true;
force force ace_scopes_overwriteZeroRange = true;
force force ace_scopes_simplifiedZeroing = false;
force force ace_scopes_useLegacyUI = false;
force force ace_scopes_zeroReferenceBarometricPressure = 1013.25;
force force ace_scopes_zeroReferenceHumidity = 0;
force force ace_scopes_zeroReferenceTemperature = 15;

// ACE 要塞
force force ace_fortify_markObjectsOnMap = 1;
force force ace_fortify_timeCostCoefficient = 1;
force force ace_fortify_timeMin = 1.5;
force force acex_fortify_settingHint = 2;

// ACE 视野距离限制器
force force ace_viewdistance_enabled = false;
force force ace_viewdistance_limitViewDistance = 8000;
force force ace_viewdistance_objectViewDistanceCoeff = 5;
force force ace_viewdistance_viewDistanceAirVehicle = 8000;
force force ace_viewdistance_viewDistanceLandVehicle = 3000;
force force ace_viewdistance_viewDistanceOnFoot = 2500;

// ACE 视野限制
force force acex_viewrestriction_mode = 3;
force force acex_viewrestriction_modeSelectiveAir = 0;
force force acex_viewrestriction_modeSelectiveFoot = 0;
force force acex_viewrestriction_modeSelectiveLand = 0;
force force acex_viewrestriction_modeSelectiveSea = 0;
force force acex_viewrestriction_preserveView = true;

// ACE 载具上锁
force force ace_vehiclelock_defaultLockpickStrength = 15;
force force ace_vehiclelock_lockVehicleInventory = true;
force force ace_vehiclelock_vehicleStartingLockState = -1;

// ACE 载具按键
force force ace_vehicles_keepEngineRunning = false;
force force ace_vehicles_speedLimiterStep = 5;

// ACE 过热
force force ace_overheating_cookoffCoef = 3;
force force ace_overheating_displayTextOnJam = true;
force force ace_overheating_enabled = true;
force force ace_overheating_heatCoef = 0.75;

A3-Antistasi/MissionDescription/params.hpp
// Real defaults are set in functions/init/fn_initParams.sqf and should match the descriptions below
// 9999 is a special value that causes that parameter to be loaded from the last save, if any
// 9998 is a reserved value and should not be used

class Params
{
    class howTo0
    {
        title = "HOW TO USE ANTISTASI PARAMETERS";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class howTo1
    {
        title = "Default values will attempt to load from the save. If not found, the value in brackets will be used instead";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class howTo2
    {
        title = "Setting any other value will override that parameter with the chosen value";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class howTo3
    {
        title = "Note that this page does NOT know the saved values";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class howToSpacer
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class loadSave
    {
        title = "Load last Persistent Save";
        values[] = {1,0};
        texts[] = {"Yes","No"};
        default = 1;
    };
    class gameMode
    {
        title = "Game Mode - Do NOT change this mid mission";
        values[] = {9999,1,2,3};
        texts[] = {"Load from save (Default: Reb vs Gov vs Inv)","Reb vs Gov vs Inv","Reb vs Gov & Inv","Reb vs Gov"};
        default = 9999;
    };
    class autoSave
    {
        title = "Enable Autosave (every X minutes)";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 9999;
    };
    class autoSaveInterval
    {
        title = "Time between autosaves (in minutes)";
        values[] = {9999,600,1200,1800,3600,5400};
        texts[] = {"Load from save (Default: 60)","10","20","30","60","90"};
        default = 9999;
    };
    class membership
    {
        title = "Enable Server Membership";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 0;
    };
    class switchComm
    {
        title = "Enable Commander Switch (highest ranked player)";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 9999;
    };
    class tkPunish
    {
        title = "Enable Teamkill Punish";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 9999;
    };
    class mRadius
    {
        title = "Max distance from HQ for tasks";
        values[] = {9999,2000,4000,6000,8000,10000,12000};
        texts[] = {"Load from save (Default: 4000)","2000","4000","6000","8000","10000","12000"};
        default = 9999;
    };
    class pMarkers
    {
        title = "Allow Friendly Player Markers";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 0;
    };
    class AISkill
    {
        title = "Mission Difficulty";
        values[] = {9999,1,2,3};
        texts[] = {"Load from save (Default: Normal)","Easy","Normal","Hard"};
        default = 3;
    };
    class unlockItem
    {
        title = "Number of the same item required to unlock";
        values[] = {9999,15,25,40,1e6};
        texts[] = {"Load from save (Default: 25)","15","25","40","1 000 000"};
        default = 9999;
    };
    class memberOnlyMagLimit
    {
        title = "Number of magazines needed for guests to be able to use them";
        values[] = {9999,10,20,30,40,50,60};
        texts[] = {"Load from save (Default: 40)","10","20","30","40","50","60"};
        default = 30;
    };
    class civTraffic
    {
        title = "Rate of Civ Traffic";
        values[] = {9999,0,1,2,4,};
        texts[] = {"Load from save (Default: Medium)","None","Low","Medium","High"};
        default = 4;
    };
    class memberSlots
    {
        title = "Percentage of Reserved Slots for Members";
        values[] = {9999,0,20,40,60,80,100};
        texts[] = {"Load from save (Default: 20%)","None","20%","40%","60%","80%","All"};
        default = 0;
    };
    class memberDistance
    {
        title = "Max distance non members can be from the closest member or HQ (they will be teleported to HQ after some timeout)";
        values[] = {9999,4000,5000,6000,7000,8000,10000,16000,-1};  // 16000 is left as backwards compatibility
        texts[] = {"Load from save (Default: 5 km)","4 km","5 km","6 km","7 km","8 km","10 km","16 km","Unlimited"};
        default = -1;
    };
    class allowFT
    {
        title = "Fast Travel Targets Allowed";
        values[] = {9999,0,1};
        texts[] = {"Load from save (Default: Airports & HQ)","Any friendly position","Only Airports & HQ"};
        default = 0;
    };
    class napalmEnabled
    {
        title = "Enable Napalm Bombing for AI";
        values[] = {9999,0,1};
        texts[] = {"Load from save (Default: No)","No","Yes"};
        default = 1;
    };
    class unlockedUnlimitedAmmo
    {
        title = "Do Unlocked Weapons Automatically Unlock Their Standard Magazine?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 1;
    };
    class allowGuidedLaunchers
    {
        title = "Should Guided Launchers become unlocked?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 1;
    };
    class allowUnlockedExplosives
    {
        title = "Should Explosives become unlocked?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 1;
    };
    class startWithLongRangeRadio
    {
        title = "[TFAR] Start with Long Range Radio?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: Yes)","Yes","No"};
        default = 9999;
    };
    class helmetLossChance
    {
        title = "Chance of helmet loss on headshots";
        values[] = {9999,0,33,66,100};
        texts[] = {"Load from save (Default: Sometimes)","Never","Sometimes","Often","Always"};
        default = 9999;
    };
    class Spacer10
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class EnableLootToCrate
    {
        title = "Enable Loot to crate";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: Enabled)","Disabled", "Enabled"};
        default = 1;
    };
    class LTCLootUnlocked
    {
        title = "Loot to crate: transfers unlocked items";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: Disabled)", "Disabled", "Enabled"};
        default = 0;
    };
    class Spacer11
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class Kart
    {
        title = "Allow Items and Vehicles from Karts DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Mark
    {
        title = "Allow Items and Vehicles from Marksman DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Heli
    {
        title = "Allow Items and Vehicles from Heli DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Expansion
    {
        title = "Allow Items and Vehicles from Apex DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Jets
    {
        title = "Allow Items and Vehicles from Jets DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Orange
    {
        title = "Allow Items and Vehicles from Laws of War DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Tanks
    {
        title = "Allow Items and Vehicles from Tanks DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class GlobMob
    {
        title = "Allow Items and Vehicles from Global Mobilization DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Enoch
    {
        title = "Allow Items and Vehicles from Contact DLC?";
        values[] = {9999,1,0};
        texts[] = {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class AoW
    {
        title = "Allow Art of War DLC?";
        values[] = {9999,1,0};
        texts[] =  {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
        class VN
    {
        title = "Allow VN DLC (S.O.G. PRAIRIE FIRE)?";
        values[] = {9999,1,0};
        texts[] =  {"Load from save (Default: No)","Yes","No"};
        default = 9999;
    };
    class Spacer0
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class AdvancedParams
    {
        title = "ADVANCED USERS ONLY";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class LogLevel
    {
        title = "Logging Level (Amount of detail in .rpt file)";
        values[] = {9999,1,2,3};
        texts[] = {"Load from save (Default: Debug)", "Error", "Info", "Debug"};
        default = 1;
    };
    class SupportOptions
    {
        title = "SUPPORT OPTIONS";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class allowUnfairSupports
    {
        title = "[Experimental - Use at own risk] Allow unfair supports to be used by the enemy";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: Not Allowed)", "Not allowed", "Allowed"};
        default = 1;
    }
    class allowFuturisticSupports
    {
        title = "[Experimental - Use at own risk](Requires unfair supports) Allow futuristic supports to be used by the enemy";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: Not Allowed)", "Not allowed", "Allowed"};
        default = 1;
    }
    class CrateOptions
    {
        title = "LOOT CRATE OPTIONS";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class truelyRandomCrates
    {
        title = "[Experimental] Truely Random Crates: Remove all balance checks from Crates";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: False)", "False", "True"};
        default = 9999;
    };
    class cratePlayerScaling
    {
        title = "Decrease loot quantity as player count increases? (Yes is recommended for balance reasons)";
        values[] = {9999, 0, 1};
        texts[] = {"Load from save (Default: True)", "False", "True"};
        default = 9999;
    };
    class crateWepTypeMax
    {
        title = "Maximum Weapon Types in Crates";
        values[] = {9999,0,2,4,6,8,12,16};
        texts[] = {"Load from save (Default: 9)","1","3","5","7","9","13","17"};
        default = 9999;
    };
    class crateWepNumMax
    {
        title = "Maximum Weapon Quantity in Crates";
        values[] = {9999,0,1,3,5,8,10,15};
        texts[] = {"Load from save (Default: 8)","None","1","3","5","8","10","15"};
        default = 9999;
    };
    class Spacer1
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateItemTypeMax
    {
        title = "Maximum Item Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 5)","1","3","5","10"};
        default = 9999;
    };
    class crateItemNumMax
    {
        title = "Maximum Item Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 5)","None","1","3","5","10","15"};
        default = 9999;
    };
    class Spacer2
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateAmmoTypeMax
    {
        title = "Maximum Ammo Types in Crates";
        values[] = {9999,0,2,4,6,9,14,19};
        texts[] = {"Load from save (Default: 7)","1","3","5","7","10","15","20"};
        default = 9999;
    };
    class crateAmmoNumMax
    {
        title = "Maximum Ammo Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15,20,25,30};
        texts[] = {"Load from save (Default: 20)","None","1","3","5","10","15","20","25","30"};
        default = 9999;
    };
    class Spacer3
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateExplosiveTypeMax
    {
        title = "Maximum Explosive Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 3)","1","3","5","10"};
        default = 9999;
    };
    class crateExplosiveNumMax
    {
        title = "Maximum Explosive Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 5)","None","1","3","5","10","15"};
        default = 9999;
    };
    class Spacer4
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateAttachmentTypeMax
    {
        title = "Maximum Attachment Types in Crates";
        values[] = {9999,0,2,4,6,9,14,19};
        texts[] = {"Load from save (Default: 7)","1","3","5","7","10","15","20"};
        default = 9999;
    };
    class crateAttachmentNumMax
    {
        title = "Maximum Attachment Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15,20,25,30};
        texts[] = {"Load from save (Default: 15)","None","1","3","5","10","15","20","25","30"};
        default = 9999;
    };
    class Spacer5
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateBackpackTypeMax
    {
        title = "Maximum Backpack Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 0)","1","3","5","10"};
        default = 9999;
    };
    class crateBackpackNumMax
    {
        title = "Maximum Backpack Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 3)","None","1","3","5","10","15"};
        default = 9999;
    };
    class Spacer6
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateVestTypeMax
    {
        title = "Maximum Vest Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 0)","1","3","5","10"};
        default = 9999;
    };
    class crateVestNumMax
    {
        title = "Maximum Vest Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 3)","None","1","3","5","10","15"};
        default = 9999;
    };
    class Spacer7
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateHelmetTypeMax
    {
        title = "Maximum Helmet Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 0)","1","3","5","10"};
        default = 9999;
    };
    class crateHelmetNumMax
    {
        title = "Maximum Helmet Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 0)","None","1","3","5","10","15"};
        default = 9999;
    };
    class Spacer8
    {
        title = "";
        values[] = {""};
        texts[] = {""};
        default = "";
    };
    class crateDeviceTypeMax
    {
        title = "Maximum Device Backpack Types in Crates";
        values[] = {9999,0,2,4,9};
        texts[] = {"Load from save (Default: 3)","1","3","5","10"};
        default = 9999;
    };
    class crateDeviceNumMax
    {
        title = "Maximum Device Backpack Quantity in Crates";
        values[] = {9999,0,1,3,5,10,15};
        texts[] = {"Load from save (Default: 3)","None","1","3","5","10","15"};
        default = 9999;
    };
};
